tests/installation/disk_boot.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2016-2018 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Summary: Boot from disk
+# Maintainer: Santiago Zarate <santiago.zarate@suse.com>
+
+use base "bootbasetest";
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures qw(is_aarch64);
+use utils qw(is_uefi_boot);
+
+sub run {
+    if (is_uefi_boot || is_aarch64) {
+        shift->wait_boot(bootloader_time => 300);
+    } else {
+        die "This test module is meant for uefi capable systems";
```

Comment:
Why? `wait_boot` is generic

Reply:
it isn't needed tbh, same as above about uefi stuff